pages.nl/linux/apt.md

```diff
@@ -0,0 +1,38 @@
+# apt
+
+> Hulpprogramma voor pakketbeheer voor op Debian gebaseerde distributies.
+> Aanbevolen vervanging voor `apt-get` bij interactief gebruik in Ubuntu versie 16.04 en later.
+> Voor gelijkwaardige commando's in andere pakket managers, zie <https://wiki.archlinux.org/title/Pacman/Rosetta>.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt` commando):
+
+`sudo apt update`
+
+- Zoek naar een bepaald pakket:
+
+`apt search {{package}}`
+
+- Toon informatie over een pakket:
+
+`apt show {{package}}`
```

Comment:
```suggestion
- Toon informatie voor een specifiek pakket:

`apt show {{pakket1 pakket2 ...}}`
```

Almost the same issue as above.
Mandatory fix (error source: original page): [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): ellipsis expected.
Note: translation source: Show information for a specific package.

Reply:
Thanks for the review! Fixed in the latest commit.